src/subcommand/wallet.rs

```diff
@@ -1,15 +1,18 @@
 use super::*;
 
 mod identify;
+mod list;
 
 #[derive(Debug, Parser)]
 pub(crate) enum Wallet {
+  List,
   Identify,
```

Comment:
These should be sorted:

```suggestion
  Identify,
  List,
```